### ISSUE TITLE
make dist ERR!Could not find base.js

### ISSUE THREAD

**Issue by gitzhang** [mapgears/ol3-google-maps#149]
ol3-google-maps git:(master) make dist
mkdir -p dist/
node build/build.js build/ol3gm.json dist/ol3gm.js
info ol3-google-maps Parsing dependencies
ERR! Could not find base.js
make: *** [dist/ol3gm.js] Error 1

**Comment by adube** [comment]
I just tried it on my Xubuntu 16.04 machine, with NodeJS v4.2.6 and it worked for me.  Would you please provide more info?

**Comment by gitzhang** [comment]
tks for reply. I tried on macOS sierra 10.12.1 with NodeJS v6.6.0. And I changed my NodeJS version to 4.4.4 , but it was still same ERR.

**Comment by adube** [comment]
I just tried it on a OS X Yosemie 10.10.5 Mac using:

* nvm (to install node.  I used version 4.7.0)
  * download nvm: https://github.com/creationix/nvm#install-script
  * install node: `nvm install 4`
  * use node: `nvm use 4`
* java jdk
  * type `java`.  If you do not have it, you will be prompted a message to install it.

From there, I cloned the repository and did `make check` and it worked.  I also tried `make serve` and it worked too.

I am not sure what is wrong on your side.  If I am able to test this with your same version of OS X, I will try again.
